docs/docs-content/release-notes/release-notes.md

```diff
@@ -28,19 +28,41 @@ tags: ["release-notes"]
   [Palette Virtual Clusters](../clusters/palette-virtual-clusters/palette-virtual-clusters.md#upgrade-virtual-clusters)
   page to learn how to migrate your virtual cluster workloads.
 
-#### Features
-
 #### Improvements
 
+- Palette and VerteX emails have improved content and updated design to ensure consistency and improve accessibility.
+  The updates have been applied to sign-up, login, password resets, and billing updates emails.
+
 #### Deprecations and Removals
 
+- Palette no longer supports for the edge virtual clusters. Several components no longer integrate with the
+  [libvirt](https://libvirt.org/) virtualization API, including the Palette UI, Terraform providers, the user interface,
+  and packs.
+
 ### Edge
 
 #### Features
 
+- <TpBadge /> Palette now allows the configuration of a primary, in-cluster registry that stores the images required for
```

Comment:
```suggestion
- <TpBadge /> Palette now allows the configuration of an in-cluster primary registry that stores the images required for
```
switches these adj around because *primary registry* is a term we made up to describe the architecture